### PR TITLE
GA of assign roles to groups

### PR DESCRIPTION
**Pull request by abhijeetsinha**

### CODE REVIEW

**Comment by jasonjoh** [comment]
Please disregard the earlier message from msftbot. Your pull request was incorrectly flagged.

File: api-reference/v1.0/api/group-post-groups.md [hunk 1]
```diff
@@ -271,6 +272,86 @@ Content-type: application/json
...
+  "isAssignableToRole": true,
+  "mailEnabled": true,
+  "securityEnabled": true,
+  "mailNickname": "contosohelpdeskadministrators",
+  "visibility" : "Private"
+}
+```
+
+> **Note:** The **visibility** and **groupTypes** properties are not required for creation, but are auto-populated with these values. A group with **isAssignableToRole** property set to `true` cannot be of dynamic membership type and cannot have an owner. For more information, see [Using a group to manage Azure AD role assignments](https://go.microsoft.com/fwlink/?linkid=2103037).
+
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/group-post-groups.md:304]:
```suggestion

The following is an example of the request. The **visibility** and **groupTypes** properties are not required for creation, but are auto-populated with these values. A group with **isAssignableToRole** property set to `true` cannot be of dynamic membership type and cannot have an owner. See [Using a group to manage Azure AD role assignments](https://go.microsoft.com/fwlink/?linkid=2103037).

<!-- {
  "blockType": "request",
  "name": "create_role_enabled_group"
}-->
``` http
POST https://graph.microsoft.com/v1.0/groups
Content-Type: application/json

{
  "description": "Group assignable to a role",
  "displayName": "Role assignable group",
  "groupTypes": [
    "Unified"
  ],
  "isAssignableToRole": true,
  "mailEnabled": true,
  "securityEnabled": true,
  "mailNickname": "contosohelpdeskadministrators",
  "visibility" : "Private"
}
```
```

File: api-reference/v1.0/resources/group.md [hunk 2]
```diff
@@ -124,6 +124,7 @@ This resource supports:
...
 |hideFromOutlookClients |Boolean |True if the group is not displayed in Outlook clients, such as Outlook for Windows and Outlook on the web; otherwise, false. Default value is **false**. <br><br>Returned only on `$select`. Supported only on the Get group API (`GET /groups/{ID}`)|
 |id|String|The unique identifier for the group. <br><br>Returned by default. Inherited from [directoryObject](directoryobject.md). Key. Not nullable. Read-only.|
+|isAssignableToRole|Boolean|Indicates whether this group can be assigned to an Azure Active Directory role or not.<br><br>This property can only be set while creating the group and is immutable. Only Global Administrator and Privileged Role Administrator roles can set this property. For more information, see [Using a group to manage Azure AD role assignments](https://go.microsoft.com/fwlink/?linkid=2103037)<br><br>Returned by default.|
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/group.md:127]: Proposed changes changes on other comments. If **no owners** is a requirement, let's also state that here.
```suggestion
|isAssignableToRole|Boolean|Indicates whether this group can be assigned to an Azure Active Directory role.<br><br>This property can only be set while creating the group and is immutable. If set to `true`, the **securityEnabled** property must also be set to `true` and the group cannot be a dynamic group (that is, **groupTypes** cannot contain `DynamicMembership`). Only callers in Privileged Role Administrator and Global Administrator roles can set the value of this property. Optional. For more, see [Using a group to manage Azure AD role assignments](https://go.microsoft.com/fwlink/?linkid=2103037). <br><br>Returned by default.|
```

File: api-reference/v1.0/api/group-post-groups.md [hunk 3]
```diff
@@ -271,6 +272,86 @@ Content-type: application/json
...
+  "visibility" : "Private"
+}
+```
+
+> **Note:** The **visibility** and **groupTypes** properties are not required for creation, but are auto-populated with these values. A group with **isAssignableToRole** property set to `true` cannot be of dynamic membership type and cannot have an owner. For more information, see [Using a group to manage Azure AD role assignments](https://go.microsoft.com/fwlink/?linkid=2103037).
+ 
+#### Response
+
+The following is an example of the response. It includes only default properties.
+
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/group-post-groups.md:308]:
```suggestion

The following is an example of the response. It includes only default properties.

>**Note:** The response object shown here might be shortened for readability.

```

**Review by FaithOmbongi — COMMENTED** [review]
Proposed a few changes.

**Comment by FaithOmbongi** [comment]
The changes in this PR are already live. They were introduced via PR [12011](https://github.com/microsoftgraph/microsoft-graph-docs/pull/12011).

Closing this PR. cc @abhijeetsinha

**Pull request closed without merging**